eole/tests/test_model_lm/config.json

```diff
@@ -10,7 +10,7 @@
       "layers": 2,
       "heads": 2
     },
-    "architecture": "transformer_lm",
+    "architecture": "custom",
```

Comment:
Why is this needed?

Reply:
it is not "NEEDED" but it's clearer since all settings are set in decoder and not at model level.
This appeared when I tried a further refactoring not duplicating all settings.